core/src/main/scala/filodb.core/memstore/TimeSeriesShard.scala

```diff
@@ -1100,6 +1100,7 @@ class TimeSeriesShard(val dataset: Dataset,
       val (_, partKeyAddr, _) = BinaryRegionLarge.allocateAndCopy(partKeyBase, partKeyOffset, bufferMemoryManager)
       val partId = if (usePartId == CREATE_NEW_PARTID) createPartitionID() else usePartId
       val newPart = if (shouldTrace(partKeyAddr)) {
+        logger.debug(s"Adding tracing TSPartition dataset=${dataset.ref} shard=$shardNum group=$group partId=$partId")
```

Comment:
Log entry already exists for this. See line 1113

Reply:
The problem is that the log from line 1113 simply does not show up.  It doesn't hurt to log more.